Sources/AirRobeWidget/Views/OptInView/AirRobeOptInView.swift

```diff
@@ -58,13 +58,13 @@ final class AirRobeOptInView: UIView, NibLoadable {
     }
 
     override func layoutSubviews() {
-        let maxWidth = subTitleContainer.bounds.width - potentialValueLabel.bounds.width - 10 - potentialValueLoading.bounds.width
-        if descriptionValueLabelMaxWidth != maxWidth && viewType == .optIn {
+        let maxWidth = subTitleContainer.bounds.width - potentialValueLabel.bounds.width - potentialValueLoading.bounds.width
```

Comment:
Perhaps a helper method here would help explain what this calculation is for?

For example:

```swift
let maxWidth = usableHorizontalSpace()
```

and having the calculation be hidden behind the helper method helps make the intention of the calculation more explicit.

Reply:
It also allows for testing that function with a unit test at some point as well

Reply:
This makes sense! 👍 
will create a helper method for this.